scss/_reboot.scss

```diff
@@ -420,24 +420,27 @@ textarea {
 }
 
 // Remove the inheritance of text transform in Firefox
-
 button,
 select {
   text-transform: none;
 }
-
 // Set the cursor for non-`<button>` buttons
 //
 // Details at https://github.com/twbs/bootstrap/pull/30562
 [role="button"] {
   cursor: pointer;
 }
 
-// Remove the inheritance of word-wrap in Safari.
-// See https://github.com/twbs/bootstrap/issues/24990
-
 select {
+  // Remove the inheritance of word-wrap in Safari.
+  // See https://github.com/twbs/bootstrap/issues/24990
   word-wrap: normal;
+
+  // Undo the opacity change from Chrome
+  &:disabled {
+    color: $input-color;
```

Comment:
I'm afraid this is not enough, see [the initial reduced test case using this PR's dist file](https://codepen.io/ffoodd/pen/VwmQPjp). We currently set `.form-select:disabled { color: … }` which overrides this rule.

I'd be in favor of dropping the `color` rule here **and** in `forms/_form-select.scss` (since we don't provide `color` for other disabled inputs).

Reply:
I tackled this by myself  to save you a few minutes, but feel free to drop my commit if that doesn't fit your idea :)